ui/packages/statement/src/components/StatementsTable.tsx

```diff
@@ -39,15 +39,15 @@ const tableColumns = (
     key: 'sum_latency',
     sorter: (a: StatementOverview, b: StatementOverview) =>
       a.sum_latency! - b.sum_latency!,
-    render: (text) => getValueFormat('ns')(text, 2, null),
+    render: text => getValueFormat('ns')(text, 2, null),
```

Comment:
You can fix these format changes by `yarn format` command.

Reply:
I think yarn format has some issues here, I ran this command many times, but CI is keeping fail, any ideas?

Reply:
Ok, let me have a look

Reply:
hi @qxhy123, I help fix the format in https://github.com/pingcap-incubator/tidb-dashboard/commit/35d215805cf70f32357d651b5272010c5edab6fe , you can cherry-pick this commit, and merge the latest code in the master branch, then the PR can be merged.